src/main/java/org/apache/mesos/state/api/StateResource.java

```diff
@@ -75,122 +69,6 @@ public Response getFrameworkId() {
-     * exist or the data couldn't be read. This may fail even if the task name is valid if
-     * TaskStatus data hadn't yet been written by the Framework.
-     */
-    @Path("/tasks/status/{taskName}")
-    @GET
-    public Response getTaskStatus(@PathParam("taskName") String taskName) {
-        logger.info("Attempting to fetch TaskInfo for task '{}'", taskName);
-
-        Optional<Protos.TaskStatus> taskStatus = stateStore.fetchStatus(taskName);
-        if (taskStatus.isPresent()) {
-            return Response.ok(new JsonFormat().printToString(taskStatus.get()),
-                    MediaType.APPLICATION_JSON).build();
-        } else {
-            // Warning instead of Error: Subject to user input
-            logger.warn(String.format("Failed to fetch requested TaskStatus for task '%s'", taskName));
-            return Response.status(Response.Status.NOT_FOUND).build();
-        }
-    }
-
-    /**
-     * Returns connection information for the given task.  The response has the following format:
-     *
-     * {
-     *     "dns": "task-name.framework-name.mesos"
-     *     "ports": "8080,2000-3000"
-     * }
-     *
-     * @param taskName Name of the task
-     * @return 200 or 404
-     */
-    @Path("/tasks/connection/{taskName}")
-    @GET
-    public Response getConnection(@PathParam("taskName") String taskName) {
-        Optional<Protos.TaskInfo> info = stateStore.fetchTask(taskName);
-        if (info.isPresent()) {
-            return Response.ok(
-                    getTaskConnection(info.get()).toString(),
-                    MediaType.APPLICATION_JSON)
-                    .build();
-        } else {
-            return Response.status(Response.Status.NOT_FOUND).build();
-        }
-    }
-
-    private JSONObject getTaskConnection(Protos.TaskInfo taskInfo) {
-        String dns = taskInfo.getName() + "." + frameworkName + ".mesos";
-
-        JSONObject conn = new JSONObject();
-        conn.put("dns", dns);
-        conn.put("ports", getPortString(taskInfo));
-        return conn;
-    }
-
-    private String getPortString(Protos.TaskInfo taskInfo) {
-        List<String> ports = new ArrayList<>();
-        for (Protos.Resource resource : taskInfo.getResourcesList()) {
-            if (resource.getName() == "ports" && resource.getType() == Protos.Value.Type.RANGES) {
-                for (Protos.Value.Range range : resource.getRanges().getRangeList()) {
-                    Long begin = range.getBegin();
-                    Long end = range.getEnd();
-                    if (begin.equals(end)) {
-                        ports.add(begin.toString());
-                    } else {
-                        ports.add(begin + "-" + end);
-                    }
-                }
-            }
-        }
-        return String.join(",", ports);
-    }
-
-
-    /**
-     * Produces a listing of the names of all stored tasks.
-     */
     @Path("/properties")
```

Comment:
The `TaskResource` constructors make reference to properties, but don't appear to actually serve property query responses.  Needs to be resolved one way or the other.  Was the intention of the `StateResource` to now solely present properties?